src/view/observer/clickobserver.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import DomEventObserver from './domeventobserver.js';
+
+/**
+ * {@link engine.view.Document#click Click} event observer.
+ *
+ * Note that this observer is not attached by the {@link engine.view.Document} and is not available by default.
```

Comment:
If we're saying this it would be much more useful if we said how then to enable this observer.